tests/Database/DatabaseSqlServerSchemaGrammarTest.php

```diff
@@ -472,7 +472,7 @@ public function testAddingEnum()
         $statements = $blueprint->toSql($this->getConnection(), $this->getGrammar());
 
         $this->assertCount(1, $statements);
-        $this->assertEquals('alter table "users" add "role" nvarchar(255) check ("role" in (\'member\', \'admin\')) not null', $statements[0]);
+        $this->assertEquals('alter table "users" add "role" nvarchar(255) check ("role" in (N\'member\', N\'admin\')) not null', $statements[0]);
```

Comment:
Modifying this test actually proves that this isn't wanted. Wouldn't this break existing apps?

Reply:
If we replace `member` and `admin` by some Chinese, and run it in SQL server, it won't work.

Reply:
It seems this is indeed valid: http://www.sqlservertutorial.net/sql-server-basics/sql-server-nvarchar/